File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{ 
 int i,j,k;
 float n;   
 prompt:
 printf("Height: ");
 scanf("%f",&n);
 if (n<1){
     goto prompt;
 }
 else if (n>8){
    goto prompt;   
}
 else { 
     for (i=0;i<=n-1;i++){
       for (j=0;j<=(n-i)-2;j++){
               printf(" ");
       }
         for (k=0;k<=i;k++){
             printf("#");
         }
      printf("\n");
     }
               }
               } 

File: mario.py
from cs50 import get_int
#..
h = 0
while h<1 or h>8:
     h = get_int("Height: ")
#..
i = 0
j = 0
k = 0
#..
for j in range(h):
    while k < h-(j+1):
        print(" ",end="")
        k = k + 1
    #...
    while i < j+1:
        print("#",end="")
        i = i + 1
    print("")
    i = 0
    k = 0

